test/plugins/features/movie/controller.test.js

```diff
@@ -13,12 +13,49 @@ describe('movie controller', () => {
       return Controller.create(payload)
       .then((movie) => {
         expect(movie.get('title')).to.eql(payload.title);
+      });
+    });
 
-        return new Movie({ id: movie.id }).fetch();
-      })
-      .then((movie) => {
-        expect(movie.get('title')).to.eql(payload.title);
-        expect(movie.get('release_year')).to.eql(payload.release_year);
+  });
+
+  describe('get', () => {
+
+    const movie = { title: 'foo', release_year: 2016 };
+
+    beforeEach(() => {
+      return Knex('movies').insert(movie);
```

Comment:
You should create 2 movies and insert them into the DB because right now, your tests just makes sure that this movie comes back. It would be better if there were 2 movies and the first test (no filters) returns both while the other two just returns one based on the appropriate filters. That way, it's more accurately testing functionally.